README.md

```diff
@@ -121,7 +135,7 @@ Promise.using(filedisk.openFile('/path/to/some/file', 'r'), function(fd) {
 		assert(buf.equals(buf2));
 	})
 	.then(function() {
-		return disk.getStreamAsync();
+		return disk.getStreamAsync(null, null);
```

Comment:
If I don't pass `null`, `null` as `position` and `length`, `getStream` would consider that the callback is `position`.